kubernetes/src/main/scala/org/apache/spark/scheduler/cluster/kubernetes/KubernetesClusterScheduler.scala

```diff
@@ -70,121 +72,50 @@ private[spark] class KubernetesClusterScheduler(conf: SparkConf)
     logInfo("Starting spark driver on kubernetes cluster")
     val driverDescription = buildDriverDescription(args)
 
-    // This is the URL of the spark distro.
-    val sparkDistUri = Option(System.getenv("SPARK_DISTRO_URI")).getOrElse {
-      throw new SparkException("Spark distribution not set, please set the SPARK_DISTRO_URI environment variable to " +
-        "a runnable spark archive.")
-    }
-
-    // This is the URL of the driver pod's image.
-    // Any image may be supplied as long as it contains a
-    // ./install.sh file which is executable and sets up the
-    // spark environment in /opt/spark.
-    val sparkDriverImage = Option(System.getenv("SPARK_DRIVER_IMG")).getOrElse {
-      throw new SparkException("Spark driver image not set, please set the SPARK_DRIVER_IMG environment variable to " +
-        "a spark driver image.")
+    // image needs to support shim scripts "/opt/driver.sh" and "/opt/executor.sh"
+    val sparkDriverImage = conf.getOption("spark.kubernetes.sparkImage").getOrElse {
```

Comment:
I think for now, we are passing this with the `--conf` flag to spark-submit. We may want to put this and similar others in a configuration file so that they are in one place.